lib/util/mp4_box_parsers.js

```diff
@@ -625,6 +625,40 @@ shaka.util.Mp4BoxParsers = class {
     };
   }
 
+  /**
+   * Parses AV1 codec string with COLR box information.
+   *
```

Comment:
It'd be nice to have some kind of reference linked for this. Would this be an appropriate citation for this method?

https://aomediacodec.github.io/av1-isobmff/#codecsparam

Reply:
Done!